DNN Platform/DotNetNuke.Abstractions/Entities/IConfigurationSetting.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Abstractions.Entities
+{
+    /// <summary>
+    /// The configuration setting.
+    /// </summary>
+    public interface IConfigurationSetting
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether if the configuration setting is secure.
+        /// </summary>
+        bool IsSecure { get; set; }
+
+        /// <summary>
+        /// Gets or sets the configuration key.
+        /// </summary>
+        string Key { get; set; }
+
+        /// <summary>
+        /// gets or sets configuraiton value.
```

Comment:
```suggestion
        /// Gets or sets the configuration setting value.
```

Reply:
Duplicate, there is another item for this further down. I'm going to resolve this conversation

https://github.com/dnnsoftware/Dnn.Platform/pull/3990#discussion_r474004736